projects/saturn/src/main/java/io/fairspace/saturn/config/Config.java

```diff
@@ -76,6 +81,20 @@ public static class Auth {
     public static class WebDAV {
         public String blobStorePath = "data/blobs";
     }
+    @Builder
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class CacheConfig {
+        public String name;
+        public boolean enabled;
+        public boolean autoRefreshEnabled;
```

Comment:
please set reasonable defaults here, since application.yaml gets fully overwritten when deploying with helm.

Reply:
It would be good to include these in helm deployment setup too

Reply:
Ddefaults value added, but did not include them in helm since not sure I understood correctly the comment. Resolving not to be blocked.